src/book/02-system/04-other/01-nubook.mdx

```diff
@@ -98,13 +96,17 @@ All additions and edits are done through GitHub pull requests. To add or edit co
 5. Commit and push your changes to GitHub.
 6. Go to the repo on GitHub and create a pull request for your branch. Your pull request will be reviewed, merged, and deployed to the live site.
 
+More information on using Git can be found on the [Git Guide](/guides/general/git).
+
 ## Writing Markdown
 
 NUbook content is written using [MDX](https://mdxjs.com/), an extension of [Markdown](https://daringfireball.net/projects/markdown/) with support for dynamic content via [JSX](https://reactjs.org/docs/introducing-jsx.html) and [React](https://reactjs.org/) components. Markdown provides a minimal syntax for writing and styling text content.
 
-GitHub has [a good guide](https://guides.github.com/features/mastering-markdown/) for getting started with Markdown. There's also [this short video series](https://www.youtube.com/playlist?list=PLu8EoSxDXHP7v7K5nZSMo9XWidbJ_Bns3) if you prefer watching.
+GitHub has [a good guide](https://guides.github.com/features/mastering-markdown/) for getting started with Markdown. There's also [a short video series](https://www.youtube.com/playlist?list=PLu8EoSxDXHP7v7K5nZSMo9XWidbJ_Bns3) on using Markdown.
+
+NUbook has a [kitchen sink](/kitchen-sink) demonstrating the various functionalities avaliable to use when writing NUbook pages.
 
-### Adding images
+#### Adding images
```

Comment:
Skipping headings (from h2 to h4 in this case) [can be troublesome](https://stackoverflow.com/a/8783227).

Didn't see the length of the TOC with all these headings included, how big was it?

Reply:
![image](https://user-images.githubusercontent.com/35280100/85992625-7f9aae80-ba38-11ea-84c4-b4b3c9d0b215.png)
Not great but if it's an issue for screen readers then we'll not skip.

Reply:
![image](https://user-images.githubusercontent.com/35280100/85992816-c7213a80-ba38-11ea-8833-33dc84ef9e2f.png)
This is a little better

Reply:
"Deploying" and "Deploy previews" previous can also be combined into "Deploys and PR Previews"

Paragraphs might need re-wording.